### PR TITLE
DAH-1672 New Translations for Educator Listings and Veterans Preferences

### PR DESCRIPTION
**Pull request by jimlin-sfgov**
https://sfgovdt.jira.com/browse/DAH-1672

### CODE REVIEW

File: app/assets/json/translations/locale-es.json [hunk 1]
```diff
@@ -454,7 +488,7 @@
                 "not_found_body": "Estimado(a) MOHCD, no puedo completar mi solicitud para {{listing_name}}. Mi dirección es {{home_address}} y el formulario me señala que hay un error. Indíqueme qué puedo hacer a continuación. Gracias, {{first_name}} {{last_name}} {{email}} {{phone_number}}",
                 "not_found_subject": "No puedo ingresar mi dirección en la solicitud en DAHLIA"
             },
-            "address_validation_duplicate_unit": "Si tiene un número de unidad o departamento, simplemente ingréselo en el casillero “Apt or Unit #” (N.° de departamento o unidad)",
+            "address_validation_duplicate_unit": "Si tiene un número de unidad o departamento, simplemente ingréselo en el casillero \"Apt or Unit #\" (N.° de departamento o unidad).",
```
Comment (jimlin-sfgov) [inline app/assets/json/translations/locale-es.json:491]: There are 3 translations keys that are not related to Educator listings or Veteran preferences, but they were part of the translation files we received from Lionbridge.

**Review by tallulahkay — APPROVED** [review]
lgtm
